collections/_version2-english/9-fork/1-basic-knowledge.md

```diff
@@ -117,3 +126,15 @@ In this case, the threshold (`275,000` REP) is exceeded in Dispute Round 20. Tha
 
 ## When does a fork end?
 The period from the start to the end of a fork is referred to as the [forking period]({{glossary_forking_period}}). The forking period ends when either 60 days have passed, or more than 50% of [all theoretical REP]({{glossary_all_theoretical_rep}}) is migrated to one of the [child universes]({{glossary_child_universe}}). Whichever child universe receives the most migrated REP by the end of the forking period becomes the [winning universe]({{glossary_winning_universe}}), the other child universes become the [losing universe]({{glossary_losing_universe}}).
+
+## What happens to the parent universe?
+When a fork starts the [parent universe]({{glossary_parent_universe}}) becomes permanently locked. In the [locked universe]({{glossary_locked_universe}}), no new [markets]({{glossary_market}}) can be created and no markets can go through the [reporting]({{glossary_report}}) process. As a result of this, no markets can be [finalized](glossary_finalized_market). (See [Restrictions on use]({{url_restrictions_on_use}}) for details.)
+
+## What happens to the child universes?
+As mentioned [above](#when-does-a-fork-end), when the [forking period]({{glossary_forking_period}}) ends, [child universes]({{glossary_child_universe}}) become the [winning universe]({{glossary_winning_universe}}) or the [losing universe]({{glossary_losing_universe}}).
+
+In terms of what you can do, there is no difference between the winning universe and the losing universe. There are no restrictions for either universe. (See [Restrictions on use]({{url_restrictions_on_use}}) for details.)
+
+However, all markets in the [parent universe]({{glossary_parent_universe}}) can only be migrated to the winning universe and the objects attached to the forking market, such as shares, open interest, unfilled orders, are also only migrated to the winning universe. (See [Migration of the objects]({{url_migration_of_the_objects}}) for details.)
+
+Additionally, [REP]({{glossary_rep}}) in the losing universe will lose its value. Because traders will not want to trade in a universe where [reporters]({{glossary_reporter}}) have lied, and [market creators]({{glossary_market_creator}}) will not want to create a market in a universe where there are no traders. In a universe without markets or trading, REP does not provide any dividends to those holding it. However, we never know if the price of REP in the losing universe will be as expected until a [fork]({{glossary_fork}}) actually occurs.
```

Comment:
This is incorrect.  The winning universe doesn't necessarily align with the truth universe.  An attacker can make a lie universe win the fork.

In reality, any Lie universes are expected to have REP be valueless and any True universes are expected for REP to have value.  Ideally, Truth == Winning, but that isn't guaranteed (other than economically).

Reply:
Thinking...

Reply:
@MicahZoltu  What do you think of this? (The changed words are bold.)

> **From an economic rationality perspective**, REP in the losing universe **are expected to** lose its value. Because traders will not want to trade in a universe where reporters have lied, and market creators will not want to create a market in a universe where there are no traders. In a universe without markets or trading, REP does not provide any dividends to those holding it. However, **the winning universe doesn't necessarily align with the truth. a false universe can be the winning universe by an attacker who is not economically rational. Ideally, the true universe should be the winning universe, but that isn't guaranteed in cases where economic rationality does not exist**.

Reply:
> From an economic rationality perspective, REP in the losing universe are expected to lose its value.

REP in a **lie** universes is expected to lose its value.  A **losing** universe is *hopefully* a lie universe, but not necessarily.

Maybe something like:
> REP in any universe whose forking market outcome was a lie is expected to lose its value, since future traders won't want to trade in a universe where reporters have a history of lying which means REP in that universe will not receive a meaningful amount of trading fees.  When the system is working correctly, and people are behaving economically rationally (profit motivated), the expectation is that the winning universe will align with the truth universe and the losing universes will align with the lying universes.

Reply:
I will read it next week. Thanks!

Reply:
Thank you for suggesting 😀 I have read the sentence you suggested and I finally understood what you meant.
I will submit it as another pull request. Because I don't know how to include this change in this pull request.

Reply:
I submitted and merged the pull request.
Pull Request: https://github.com/msagansk/Augur.Guide/pull/31